conda/conda_build_config.yaml

```diff
@@ -1,13 +1,11 @@
 python:
   - 3.10
-  - 3.8
 numpy:
   - 1.22
-  - 1.20
 rubin_sim:
-  - 0.10
-  - 0.8
+  - 0
 zip_keys:
   - python
   - numpy
   - rubin_sim
+  
```

Comment:
Add new empty line please!

Reply:
Ooops, grabbed this as part of [7485a63]